contrib/ay-openqa-worker.xml

```diff
@@ -7,6 +7,14 @@
     </mode>
   </general>
   <networking>
+    <!-- For multiple network interfaces by default the last is configured, prefer the first -->
```

Comment:
This is just a guess of course. I've just used https://w3.nue.suse.com/~okurz/ay-openqa-worker-leap.xml which has this config as well and it didn't work on https://qesapworker-prg4-mgmt.qa.suse.cz (but I was able to select the right network interface manually and then continue with the automatic installation).

Reply:
I don't think it's a "guess" to autoyast but more like a "hint" because I observed on a machine with three ethernet interfaces that it only configured the last, unconnected one until I added this setting when the first but also the other interfaces were configured :shrug:

Reply:
I can just say that it didn't work for me. However, if it increases the likelihood for things to work in general I'm fine with it.